spec/import_spec.rb

```diff
@@ -112,21 +132,30 @@ def upload_file!(name, ext='csv')
         end
       end
 
-      context "Win1251" do
-
-        before do
-          upload_file!(:authors_win1251_win_endline)
-        end
+      context "auto detect encoding" do
+        include_examples 'successful inserts' , :auto,  :authors_win1251_win_endline
```

Comment:
Line is too long. [84/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.